### PR TITLE
feature/Initial-UI-setup

### PR DESCRIPTION
**Pull request by Saafine**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (allgreed) [inline .gitignore:1]: Please use [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for private tool artifacts

File: ui/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
+# Editor directories and files
+.idea
```
Comment (allgreed) [inline ui/.gitignore:17]: Please use [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for private tool artifacts

File: ui/.gitignore [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+node_modules
+/dist
+
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
```
Comment (allgreed) [inline ui/.gitignore:11]: Which one of those are you actually using? Could you please remove the others?

File: ui/.gitignore [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.DS_Store
+node_modules
+/dist
+
+
+# local env files
+.env.local
```
Comment (allgreed) [inline ui/.gitignore:7]: Are you actually using those? I don't see a `.env`

File: ui/.gitignore [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.DS_Store
```
Comment (allgreed) [inline ui/.gitignore:1]: I hereby accept the fact that you're using MacBook, would you kindly move this to global gitignore? ^^

File: ui/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# clustogram
```
Comment (allgreed) [inline ui/README.md:1]: I highly recommend sticking to some form of [my README template](https://github.com/allgreed/templates/blob/master/README.md_template), however I'm not having super strong opinion about this. Just please make sure that relevant `Makefile` targets work. It's easiest to achieve the make <-> README parity by simply having README redirect to make, hence my suggestion. But as long as Make works as expected you do you ;)

**Review by allgreed — CHANGES_REQUESTED** [review]
Nice work, and thank you for the first contribution to this project. I may seem nitpicky, but it's mostly to set standards and provide a case study for your teammates. I believe the comments can be addressed in less than 15 minutes and then I'll gladly merge it.

Also: could you please please provide working `default.nix` in the next few iterations? It's a soft requirement for getting all components integrated.

**Comment by Saafine** [comment]
> Nice work, and thank you for the first contribution to this project. I may seem nitpicky, but it's mostly to set standards and provide a case study for your teammates. I believe the comments can be addressed in less than 15 minutes and then I'll gladly merge it.
> 
> Also: could you please please provide working `default.nix` in the next few iterations? It's a soft requirement for getting all components integrated.

Thank you for the comments,
1) default.nix should be now fixed.
2) I updated my global .gitignore
3) I have a question to Makefile:

Do you think we need these commands too? I am not exactly sure what to put there:

`env-up: ## set up dev environment
	@echo "Not implemented"; false

env-down: ## tear down dev environment
	@echo "Not implemented"; false

env-recreate: env-down env-up ## deconstruct current env and create another one

container: build ## create container
	#docker build -t lmap .
	@echo "Not implemented"; false
`

File: ui/README.md (hunk 6 above)
Comment (Saafine) [reply ui/README.md]: I removed README from ui folder. I think it's not necessary if README in root directory exists already.

File: ui/.gitignore (hunk 5 above)
Comment (Saafine) [reply ui/.gitignore]: Done

File: ui/.gitignore (hunk 4 above)
Comment (Saafine) [reply ui/.gitignore]: Done

File: ui/.gitignore (hunk 3 above)
Comment (Saafine) [reply ui/.gitignore:5]: Removed

File: ui/.gitignore (hunk 2 above)
Comment (Saafine) [reply ui/.gitignore]: Done, good idea

File: .gitignore (hunk 1 above)
Comment (Saafine) [reply .gitignore]: Done

**Review by allgreed — APPROVED** [review]

**Pull request merged**